exchange/bidder.go

```diff
@@ -178,6 +199,64 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 	return seatBid, errs
 }
 
+func addNativeTypes(bid *openrtb.Bid, request *openrtb.BidRequest) (*nativeResponse.Response, []error) {
+	var errs []error
+	var nativeMarkup *nativeResponse.Response
+	if err := json.Unmarshal(json.RawMessage(bid.AdM), &nativeMarkup); err != nil {
```

Comment:
Why is it necessary to cast the message to `json.RawMessage` here and below on line 215?

Reply:
... I didn't see your answer above. Doh!